src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -154,3 +154,13 @@ <h2>Datepicker with custom header</h2>
     <mat-datepicker #customHeaderPicker [calendarHeaderComponent]="customHeader"></mat-datepicker>
   </mat-form-field>
 </p>
+
+<h2>Datepicker with custom header using standard header</h2>
+<p>
+  <mat-form-field>
```

Comment:
Can you create a unit test version of this that just tests that the component can be constructed without errors and the default header is in the DOM?

Reply:
I guess `src/lib/datepicker/datepicker.spec.ts` would be a good place for that. I could move my test component `CustomHeader2` there and create a test case where it is used as an input for the header.

Reply:
Yeah that sounds good, you can leave it in the demo app too if you want, but the unit test is most important